### PR TITLE
Fix asyncio.get_running_loop() monkey-patching (#83)

### PR DESCRIPTION
**Pull request by niklasf**
Since Python 3.7 `asyncio.get_running_loop()` is public, so we will have to monkey-patch that in addition to the already monkey-patched internal `asyncio.events.get_running_loop()`.

Part of #83.

### CODE REVIEW

**Comment by oremanj** [comment]
Looks good. Please add `newsfragments/99.bugfix.rst` describing the user-visible impact of this change, e.g., "`asyncio.get_running_loop()` now knows how to return the running trio-asyncio event loop."

**Comment by niklasf** [comment]
News fragment added :+1:

**Comment by shamrin** [comment]
I've tried rerunning the checks a few times, but at least one macOS test hangs due to GitHub Actions bug. Every time a different macOS hangs. For this reason I can't merge this PR :(

@oremanj could you please mark all macOS tests as non-required?

File: newsfragments/99.bugfix.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+:func:`asyncio.get_running_loop()` will now return the trio-asyncio event loop
```
Comment (oremanj) [inline newsfragments/99.bugfix.rst:1]: I believe `:func:` wants its referent to be specified without the trailing parens. This made me realize we aren't testing newsfragments as part of the docs build. I created #102 to fix.

Comment (oremanj) [reply newsfragments/99.bugfix.rst:1]: Never mind, it appears to work either way!

**Pull request merged**